lib/core/src/Ouroboros/Network/Client/Wallet.hs

```diff
@@ -116,6 +123,8 @@ import Ouroboros.Network.Protocol.LocalTxSubmission.Type
 import qualified Data.List.NonEmpty as NE
 import qualified Ouroboros.Network.Protocol.ChainSync.ClientPipelined as P
 import qualified Ouroboros.Network.Protocol.LocalStateQuery.Client as LSQ
+import qualified Data.Text as T
+import Data.Text (Text)
```

Comment:
This import should be moved to the section just above containing other unqualified imports.

(https://input-output-hk.github.io/adrestia/code/Coding-Standards#stylish-haskell-is-used-to-format-grouped-imports--language-pragmas )